lib/OpenQA/Schema/Result/ScheduledProducts.pm

```diff
@@ -319,7 +319,7 @@ Internal function to actually schedule the ISO, see schedule_iso().
 =cut
 
 sub _schedule_iso {
-    my ($self, $args) = @_;
+    my ($self, $args, $guard) = @_;
```

Comment:
I recommend using signatures in new code.

Reply:
```suggestion
```